realm/realm-library/src/main/java/io/realm/MixedNativeFunctionsImpl.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.realm;
+
+import io.realm.internal.MixedNativeFunctions;
+import io.realm.internal.Table;
+import io.realm.internal.TableQuery;
+import io.realm.internal.objectstore.OsObjectBuilder;
+
+
+public class MixedNativeFunctionsImpl implements MixedNativeFunctions {
```

Comment:
As already pointed out. What is the purpose of having this a class instead of just static methods?